lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -124,6 +125,8 @@ class OptimizedImages extends Gatherer {
         const image = {failed: false, ...stats, ...record};
         results.push(image);
       } catch (err) {
+        log.warn('optimized-images', err.message);
```

Comment:
log.verbose seems like it might be better here. These happen all the time and I think it's going to get really old seeing them :)

Maybe we need to invest more in devtools debugging? It seems like this wouldn't be an issue if it were easier to get breakpoints in DT workers

Reply:
Why would they happen all the time? I only know about the case I mentioned in the OP.